tutorials/tech-terms/index.md

```diff
@@ -0,0 +1,279 @@
+
+The server also has a stub to interface with the remote procedure.
+
+</HighlightBox>
+
+2. The client stub packs the procedure parameters into a message.
+
+<HighlightBox type="info">
+
+**Packing procedure parameters is called marshaling.**
+
+**Marshaling** is _the process of gathering data from one or more applications, putting data pieces into a message buffer, and organizing the data into a prescribed data format_.
+
+Marshaling is vital to pass output parameters of a program written in one language as inputs to programs in a different language.
+
+</HighlightBox>
+
+3. The client stub then makes a system call to send the message.
+4. The client's local operating system (OS) sends the message from the client (machine) to the server (machine) through the corresponding transport layers.
+5. Then the server OS passes the incoming packets to the server stub.
+5. The server stub unpacks the message and with it the included procedure parameters - this is called unmarshaling.
+6. The server stub calls a server procedure and the procedure is executed.
+7. Once the procedure is finalized, the output is returned to the server stub.
+8. The server stub packs the return values into a message.
+9. The message is sent to the transport layer, which sends the message to the client's transport layer.
+10. The client stub unmarshals the return parameters.
+
+<HighlightBox type="note">
+
+In an Open Systems Interconnection (OSI) model, RPC touches the transport and application layer.
+
+The transport layer is tasked with the reliable sending and receiving of messages across a network. It requires error-checking mechanisms, data flow controls, data integrity assurance, congestion avoidance, multiplexing, and same order delivery.
+
+The application layer is tasked with ensuring effective communication between applications on different computer systems and networks. It is a component of an application controlling methods to communicate with other devices.
+
+</HighlightBox>
+
+### RPC and Cosmos
+
+In Cosmos, RPCs are used by the command-line interface (CLI) among others to access chains. A node exposes several endpoints - gRPC, REST, and Tendermint endpoint.
+
+The Tendermint RPC endpoint is an HTTP1.1 server exposed by Tendermint. The default port is `26657`. The gRPC server default port is `9090`, and the REST server default port is `1317`.
+
+The Tendermint RPC is independent of the Cosmos SDK and can be configured. It uses HTTP `POST` and JSON-RPC 2.0 for data encoding.
+
+<HighlightBox type="tip">
+
+For more information on the Tendermint RPC, gRPC, and REST server, a closer look at the [Cosmos SDK documentation](https://docs.cosmos.network/master/core/grpc_rest.html).
+
+</HighlightBox>
+
+Cosmos exposes both the Tendermint RPC and the Cosmos LCD. Now, dive into LCD and specifically the Cosmos LCD.
```

Comment:
> Now, dive into LCD and specifically the Cosmos LCD.

But there is no more information on LCD coming right?

Reply:
No, just that. Unless you have the feeling we should add in more.

Reply:
Will reflect in wording more strongly that there is no more information coming.